tests/csm/rest/test_query_deployment.py

```diff
@@ -266,7 +267,7 @@ def test_45674(self):
         assert result, err_msg
         self.log.info("##### Test ended -  %s #####", test_case_name)
 
-    @pytest.mark.skip("Test not ready")
+    #@pytest.mark.skip("Test not ready")
```

Comment:
can we remove, instead of comment if done ?

Reply:
done